pkg/vz/vz_driver_darwin.go

```diff
@@ -29,6 +30,11 @@ type LimaVzDriver struct {
 	machine *virtualMachineWrapper
 }
 
+func init() {
+	//Without this the call to StartGraphicApplication fails. This has to be called before the vz cgo loads
+	runtime.LockOSThread()
```

Comment:
Though this is done in vz driver init() it will be applied even for hostagent of qemu instance.

Not sure if this will have any impact. I have done some basic testing all looked fine only. Will try to do some perf test.

Reply:
I don't think so, reverse-ssfs will be via separate process right ??

From my understanding, this LockOSThread should mostly affect go routines within hostagent.

Reply:
This is also done